Examples/Color Puddles/main.js

```diff
@@ -39,7 +39,7 @@ touchLayer.touchEndedHandler = touchLayer.touchCancelledHandler = function(touch
 }
 
 var h = new Heartbeat({handler: function() {
-	for (layer in touchLayers) {
+	for (var layer in touchLayers) {
```

Comment:
`for (layer in touchLayers)` would put `layer` into the global context: `window`. `var` ensures that it's only declared locally.

*Sort of off-topic:*
Although `var` "suffers" from hoisting. `let` is actually preferred since ES6.